Dockerfile

```diff
@@ -12,6 +12,12 @@ FROM python:${PYTHON_VERSION} as run
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV ENGINE="default"
+ENV NAME="default"
+ENV USER="default"
+ENV PASSWORD="default"
+ENV HOST="default"
+ENV PORT="default"
```

Comment:
It is not necessary to define ENV's in the Dockerfile if you set them inside the docker-compose config.